Lib/gftools/utils.py

```diff
@@ -513,10 +513,11 @@ def remove_url_prefix(url):
 
 def primary_script(ttFont, ignore_latin=True):
     g = classifyGlyphs(lambda uv:list(ftunicodedata.script_extension(chr(uv))), ttFont.getBestCmap(), gsub=ttFont.get("GSUB"))
-    if "Zyyy" in g:
-        del g["Zyyy"]
+    badkeys = [key for key in g.keys() if key.startswith("Z")]
```

Comment:
I think it should only skip `Zinh`, `Zyyy` and `Zzzz`, but not `Zanb`.

![Screenshot from 2023-09-01 14-13-11](https://github.com/googlefonts/gftools/assets/213676/b32f4a43-755d-470a-9fa8-e859e3c119bd)